tests/test_utils.py

```diff
@@ -286,3 +290,119 @@ def test_get_context_local_non_regionless_gets_local_region(
 )
 def test_get_host_region(host, expected):
     assert get_host_region(host) == expected
+
+
+def test_get_ui_parameters_no_params():
+    connection = MagicMock()
+    cursor = MagicMock()
+    connection.execute_string.return_value = (None, cursor)
+    cursor.fetchall.return_value = []
+    assert get_ui_parameters(connection) == {}
+
+    assert connection.execute_string.has_calls([])
```

Comment:
i don't think this assert can ever fail, it checks that an empty list is part of the set of calls that `execute_string`, which is always true

Reply:
sorry this test was left here by mistake - I removed it - there is a proper test under it.